src/controllers/index.js

```diff
@@ -1,9 +1,10 @@
 const express = require('express');
-
+const signup = require('./signup');
 const router = express.Router();
 
 router.get('/', (req, res) => {
```

Comment:
why ?? 
I think that you should render the home page on src/controllers/home.js file then export it here.
and you should include it with css, js, layout ... etc files.
and where is the test for the home router?

Reply:
this is for testing that the views are working 
also our task about render signup page